Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -1450,22 +1450,21 @@ PhysicalParticleContainer::AddPlasmaFlux (PlasmaInjector const& plasma_injector,
         amrex::ParallelForRNG(overlap_box, [=] AMREX_GPU_DEVICE (int i, int j, int k, amrex::RandomEngine const& engine) noexcept
         {
             const IntVect iv(AMREX_D_DECL(i, j, k));
+            amrex::ignore_unused(j,k);
+
             auto lo = getCellCoords(overlap_corner, dx, {0._rt, 0._rt, 0._rt}, iv);
             auto hi = getCellCoords(overlap_corner, dx, {1._rt, 1._rt, 1._rt}, iv);
 
             if (flux_pos->overlapsWith(lo, hi))
             {
                 auto index = overlap_box.index(iv);
-                int r;
+                int r = 1;
                 if (fine_overlap_box.ok() && fine_overlap_box.contains(iv)) {
                     r = compute_area_weights(lrrfac, flux_normal_axis);
```

Comment:
There is another small clean up that can be done. The `lrrfac` is not necessary since `rrfac` is already locally declared within the routine.

Reply:
Sounds good, thanks. I added this in my latest commit.